docs/src/scenarios/scenario4.md

```diff
@@ -58,47 +58,157 @@ tdays = (tend - tstart).value
 ## Generate the Model and Dataset
 
 ```@example scenario4
-function formSEIISRD()
-    SEIRHD = LabelledPetriNet([:S, :E, :I, :IS, :R, :D],
-                              :expo => ((:S, :I) => (:E, :I)),
-                              :conv => (:E => :I),
-                              :rec => (:I => :R),
-                              :test => (:I => :IS),
-                              :leave => (:IS => :R),
-                              :death => (:I => :D))
-    return SEIRHD
+using AlgebraicPetri
+using ModelingToolkit
+using Catlab
+using Catlab.CategoricalAlgebra, Catlab.Programs.RelationalPrograms, Catlab.WiringDiagrams.WiringDiagramAlgebras
+using ASKEM.Dec2022Demo: typed_stratify
```

Comment:
How can we install `ASKEM`?

Reply:
https://github.com/SciML/EasyModelAnalysis.jl/pull/104